.github/workflows/check_extensions.yml

```
@@ -0,0 +1,47 @@
+name: Update extension list
+
+on:
+  workflow_dispatch: # Allow running on-demand
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '6 6 * * *' # every sunday
+  pull_request:
+    paths:
+      - '.github/workflows/check_extenstions.yml'
```

couldn't figure out why this wasn't running in the CI...
realized it was the typo in the filename
```suggestion
      - '.github/workflows/check_extensions.yml'
```